.github/workflows/lint.yml

```diff
@@ -10,4 +10,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v2
```

Comment:
thanks for this PR, can you bump this to use the release `v3.2.0` and for that, you will need to set up another action which is the go one

```yaml
- uses: actions/setup-go@v3
   with:
     go-version: 1.17
     check-latest: true
```